Cargo.lock

```diff
@@ -1712,6 +1826,15 @@ version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0ee12e4891ab3acc2d95d5023022ace22020247bb8a8d1ece875a443f7dab37d"
 
+[[package]]
+name = "unicase"
+version = "2.6.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "50f37be617794602aabbeee0be4f259dc1778fabe05e2d67ee8f79326d5cb4f6"
+dependencies = [
+ "version_check",
```

Comment:
It'd be better to use autocfg here... Probably needs a PR to unicase.

Reply:
There is an open PR [here](https://github.com/seanmonstar/unicase/pull/40).

The last activity was your comment in November, so I'm not sure it's worth waiting on.